src/encoded/static/components/form.js

```diff
@@ -688,7 +688,7 @@ export class Field extends UpdateChildMixin(React.Component) {
                 searchBase={`?mode=picker&type=${schema.linkTo}`}
             />);
         } else if (schema.type === 'boolean') {
-            input = <input type="checkbox" {...inputProps} />;
+            input = <input type="checkbox" {...inputProps} checked={!!inputProps.value} />;
```

Comment:
Could we use just 'value' instead of 'inputProps.value' here? I think 'value' is defined as 'inputProps.value' on line 617.

Reply:
Done.